cmd/controller-manager/app/options/options.go

```diff
@@ -50,7 +50,7 @@ const (
 	defaultLeaderElectionNamespace      = "kube-system"
 )
 
-var defaultOSBAPIPreferredVersion = osb.Version2_12().HeaderValue()
+var defaultOSBAPIPreferredVersion = osb.LatestAPIVersion().HeaderValue()
```

Comment:
minor thing but I don't want to mislead people, we should just call this "OSBAPIVersion"